target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -199,6 +199,9 @@ hc5861)
 	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wlan5g" "rai0"
 	ucidef_set_led_default "turbo" "turbo" "$board:blue:turbo" "0"
 	;;
+hc5962)
+	ucidef_set_led_timer "status" "status" "$board:white:status" "1000" "1000"
```

Comment:
Please, use then only the white LED for `status_led` in `/etc/diag.sh` instead of the red one and don't add anything else in `/etc/board.d/01_leds`. Using red LED could be misleading as this color is usually used to indicate some error and/or problem.

We have a common/general approach over whole code, for all targets, for status LED and there is no need to reproduce vendor firmware approach here.

Thanks!